_docs/Definition-of-Done.md

```diff
@@ -0,0 +1,21 @@
+# Definition of Done
+
+An issue can be considered "done" when the following criteria are met:
+
+1. The code compiles
+2. Unit tests have been created or modified for the modified code
+3. All unit tests pass
+4. The change has been reviewed by another team member
+5. No high priority bugs are outstanding arising from that issue
+6. The code has been merged into `main` branch
+7. Tina Cloud has been updated if affected
```

Comment:
```suggestion
7. TinaCloud has been updated if affected
```

Reply:
Though I personally prefer no-space, the system has "Tina Cloud" throughout its documentation, its repo name and the actual https://app.tina.io.

Reply:
So there are lots of other places to fix up the name - the official name is TinaCloud